Makefile

```diff
@@ -0,0 +1,10 @@
+CC = gcc
```

Comment:
This is a Makefile that is used to automatically compile your code. You can simply type `make` now in your command-line and it will run gcc... for you.